src/get-static-value.ts

```diff
@@ -1,8 +1,23 @@
-/* globals globalThis, global, self, window */
-
-import { findVariable } from "./find-variable.mjs"
-
-const globalObject =
+/* globals globalThis, global */
+/* eslint
+     @eslint-community/mysticatea/eslint-comments/no-use:0,
+     @eslint-community/mysticatea/ts/no-unsafe-assignment:0,
+     @eslint-community/mysticatea/ts/no-unsafe-argument:0,
+     @eslint-community/mysticatea/ts/no-unsafe-return:0,
+     @eslint-community/mysticatea/ts/no-unsafe-member-access:0,
+     @eslint-community/mysticatea/ts/no-unsafe-call:0,
+   ---
+   The logic within this source makes heavy use of any.
+*/
+import type { Scope } from "eslint"
+import type * as ESTree from "estree"
+import { findVariable } from "./find-variable"
+
+type AnyFunction = (...any: any[]) => any
+declare const self: typeof globalThis
+declare const window: typeof globalThis
+
+const globalObject: Record<string, any> =
```

Comment:
We should use `unknown` instead of `any`
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any

```suggestion
const globalObject: Record<string, unknown> =
```